src/openforms/registrations/contrib/objects_api/tests/test_utils.py

```diff
@@ -0,0 +1,36 @@
+from django.test import TestCase
+
+from ..utils import escape_html_manually
+
+
+class EscapeHTMLTests(TestCase):
+    def test_given_dict_values_are_escaped(self):
+        sample = {
+            "test1": "normal value",
+            "test2": "<script>alert();</script>",
```

Comment:
maybe the test input can be a bit more aimed at the html entities involved:

```suggestion
            "test2": """<>'"&""",
```
so that covers the characters <kbd><</kbd>, <kbd>></kbd>, <kbd>&</kbd>, <kbd>'</kbd>, <kbd>"</kbd>, which are HTML-unsafe